binsync/ui/panel_tabs/util_panel.py

```diff
@@ -14,6 +16,7 @@
     QLineEdit,
     QIntValidator
 )
+from PyQt5.QtCore import QThread, QObject, pyqtSignal
```

Comment:
Check how other files do Qt imports in binsync. You should be using the `from binsync.ui.qt_objects import ...`, it is how we normalize for multiple versions of Qt.